src/Symfony/Component/Security/Http/Tests/HttpUtilsTest.php

```diff
@@ -144,7 +144,7 @@ public function testCheckRequestPath()
         // Plus must not decoded to space
         $this->assertTrue($utils->checkRequestPath($this->getRequest('/foo+bar'), '/foo+bar'));
         // Checking unicode
-        $this->assertTrue($utils->checkRequestPath($this->getRequest(urlencode('/вход')), '/вход'));
+        $this->assertTrue($utils->checkRequestPath($this->getRequest('/'.urlencode('вход')), '/вход'));
```

Comment:
I think this test only meant to check unicode and not an encoded slash. Should I just leave it failing?

Also it's an interesting situation here: urlencoded slash could be allowed in requests. Then it is not a path separator but part of path segment.
But `HttpUtils::checkRequestPath` implementation cannot distinguish between `/x/` and `/x%2F`.